lib/viem/getTokens.ts

```
@@ -0,0 +1,70 @@
+        acc[network].push(item);
+        return acc;
+      },
+      {},
+    );
+    const promise = Object.entries(groupedFeedByNetwork).map(
+      async ([chainId, collections]) => {
+        const publicClient = getPublicClient(parseInt(chainId, 10));
+        const returnValues: ExtendedCollectionWithTokenId[] = [];
+        const calls = collections.map((c: CollectionExtended) => {
+          const subCalls = [];
+          for (let i = c.nextTokenId - 1; (i = 1); i--) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)